packages/system/kubeovn-webhook/templates/deployment.yaml

```
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "namespace-annotation-webhook.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "namespace-annotation-webhook.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "namespace-annotation-webhook.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "namespace-annotation-webhook.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      serviceAccountName: {{ include "namespace-annotation-webhook.fullname" . }}
+      securityContext:
+        runAsNonRoot: false
+      containers:
+        - name: {{ include "namespace-annotation-webhook.name" . }}
+          image: "{{ .Values.image }}"
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+          resources:
+            requests:
+              cpu: 10m
+              memory: 32Mi
```

```suggestion
              memory: 64Mi
```
The binary itself is 45Mi, so 32Mi is way too low